crates/router/src/connector/gpayments/transformers.rs

```diff
@@ -68,7 +76,6 @@ impl TryFrom<&GpaymentsRouterData<&types::authentication::PreAuthNVersionCallRou
     }
 }
 
-//TODO: Fill the struct with respective fields
 #[derive(Default, Debug, Serialize, Deserialize, PartialEq)]
 pub struct GpaymentsErrorResponse {
```

Comment:
This one can be removed

Reply:
this one is added by u and i gonna remove that enum and i gonna use this errorstruct

Reply:
Yeah this struct was created during template code generation. But we are not using this anywhere. So I think it is better to get rid of this.

Reply:
We are already using `struct TDS2ApiError` for deserializing error response.

Reply:
Yeah i removed in updated code